modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -474,6 +476,70 @@ else if ("bearer".equals(securityScheme.getScheme())) {
+                    if (openapiPathname != null && openapiPathname != pythonPathname) {
+                        LOGGER.info("Path '" + pythonPathname + "' is not consistant with the original OpenAPI definition. It will be replaced back by '" + openapiPathname + "'");
+                        paths.remove(pythonPathname);
+                        paths.put(openapiPathname, path);
+                    }
+                }
+
+                Map<HttpMethod, Operation> operationMap = path.readOperationsMap();
+                if (operationMap != null) {
+                    for (HttpMethod method : operationMap.keySet()) {
+                        Operation operation = operationMap.get(method);
+                        if (operation.getParameters() != null) {
+                            for (Parameter parameter: operation.getParameters()) {
+                                Map<String, Object> parameterExtensions = parameter.getExtensions();
+                                if (parameterExtensions != null) {
+                                    // Get and remove the (temporary) vendor extension
+                                    String swaggerParameterName = (String) parameterExtensions.remove("x-python-connexion-openapi-name");
+                                    if (swaggerParameterName != null) {
+                                        String pythonParameterName = parameter.getName();
+                                        if (swaggerParameterName != pythonParameterName) {
+                                            LOGGER.info("Reverting name of parameter '" + pythonParameterName + "' of operation '" + operation.getOperationId() + "' back to '" + swaggerParameterName + "'");
+                                            parameter.setName(swaggerParameterName);
+                                        } else {
+                                            LOGGER.debug("Name of parameter '" + pythonParameterName + "' of operation '" + operation.getOperationId() + "' was unchanged.");
+                                        }
+                                    } else {
+                                        LOGGER.debug("x-python-connexion-openapi-name was not set on parameter '" + parameter.getName() + "' of operation '" + operation.getOperationId() + "'");
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            // Sort path names after variable name fix
+            List<String> recoveredPathnames = new ArrayList(paths.keySet());
+            Collections.sort(recoveredPathnames);
+            for (String pathname: recoveredPathnames) {
+                PathItem pathItem = paths.remove(pathname);
+                paths.put(pathname, pathItem);
+            }
+        }
+        //objs.put("openAPI", openAPI);
```

Comment:
@tomghyselinck thanks for the PR, which looks good to me.

One minor suggestion: shall we remove line 541 and 486, which are commented out already?

Reply:
I'll do that in a separate PR when I've time. Thanks again for the PR.